### DISCUSSION TITLE
peacock integrated into vulpes minora keyboard build

### DISCUSSION THREAD

**Discussion by sadekbaroudi** [george-norton/peacock#2] [Show and tell]
### Summary
I built a case that incorporates the peacock into a split [vulpes minora](https://github.com/sadekbaroudi/vulpes-minora) build using xivik controllers

The case can be found here:
https://github.com/sadekbaroudi/vulpes-minora/blob/master/case/mx/vmin-mx-vik-peacock.stl

The firmware is a work in progress, found here:
https://github.com/sadekbaroudi/qmk_firmware/tree/fp/peacock

It will eventually be merged into the main `develop_fingerpunch` branch, and then `master` branch:
https://github.com/sadekbaroudi/qmk_firmware/tree/develop_fingerpunch

#### Updated final build
![image](https://github.com/george-norton/peacock/assets/299248/6e93cbd7-5100-459e-888d-8fe1e4f2e593)

Watch a video of it in action here:
https://github.com/george-norton/peacock/assets/299248/8ca564ae-5d54-463c-b998-e8ba852addc0

#### Info for final build

The peacock vulpes minora build is complete, with nice functioning firmware!
https://github.com/sadekbaroudi/qmk_firmware/commit/cd894b0fbda2531fbed3118bfb5ded42017bf939

I decided to 3D model a cover for the trackpad that also acted as a solid PLA trackpad cover... no vinyl... just print and screw in on top! You can find that here:
https://github.com/sadekbaroudi/vulpes-minora/blob/master/case/mx/vmin-vik-peacock-cover.stl

With this PLA cover (1.6mm thick), the ideal settings were:
```c
#define MXT_GAIN 4
#define MXT_TOUCH_THRESHOLD 12
```

#### Original build
![image](https://github.com/george-norton/peacock/assets/299248/d45a6965-182b-4d43-a308-108afe59b3ef)
![image](https://github.com/george-norton/peacock/assets/299248/b3098baa-5729-451a-ae5a-5e95a9d2f968)
![image](https://github.com/george-norton/peacock/assets/299248/8b09529d-4f26-4ec4-b947-701663895c03)
![image](https://github.com/george-norton/peacock/assets/299248/f594e6a7-944e-48b5-b7f9-daa2f15cf067)

**Comment by yassinebkr** [comment]
this is dope

**Reply by sadekbaroudi** [reply]
Thank you!
